.eslintrc

```diff
@@ -16,7 +16,7 @@
     "max-len": [2, 80, 4, { "ignoreUrls": true, "ignorePattern": "\\brequire\\([\"']|eslint-disable" }],
     "no-alert": 2,
     "no-array-constructor": 2,
-    "no-console": 2,
+    "no-console": 0,
```

Comment:
Do you mean that our production code uses console? (This is a typical rule in eslint due to prevalence of engineers leaving console.log()s in their code from debugging)
How many places does the production code use console?

Reply:
Aha, that makes sense. As far as I saw the production code does not use console, I mainly saw this used in Gulpfile, which is local. My next commit can revert this.